### ISSUE TITLE
Pile of poo stinks up adjacent spaces

### ISSUE THREAD

**Issue by ambihelical** [i3/i3#2052]
I'm using title_format as follows:

for_window [class="Thunderbird"] title_format "💩 %title"

This is rendered like this:
![screenshot from 2015-11-01 10-59-20](https://cloud.githubusercontent.com/assets/8704230/10870771/c00a5956-8087-11e5-8363-ee5eb3ef21da.png)

Interestingly, only the adjacent space characters are rendered as blocks, as far as I can tell, and rendering returns to normal after some other character.  I tried the following variationson on the text between the poo character and %title:

" " (one space) -> block
"  " (two spaces) -> two blocks
"x" (non-space) -> renders correctly
" x"  (space followed by non-space) -> block followed by correct render
" x " (space, non-space,  space) -> 1st space is block, rest is rendered correctly

As a data point, I've tried a handful of characters with unicode code points less than 0x10000, none of them have the problem.  "PILE OF POO" is U+1F4A9, it and a few others >= 0x10000 seem to exhibit this problem, but I'm not sure if that is the criteria that makes the rendering fail. 

Log: http://logs.i3wm.org/logs/5737611108810752.bz2
Version: i3 version 4.11 (2015-09-30, branch "4.11") © 2009 Michael Stapelberg and contributors

**Comment by i3bot** [comment]
I don’t see a link to logs.i3wm.org. Did you follow http://i3wm.org/docs/debugging.html? (In case you actually provided a link to a logfile, please ignore me.)

**Comment by stapelberg** [comment]
i3 does not interpret UTF-8, it gets passed through to pango/cairo directly. Can you check other applications (notably gnome-specimen) to see how they react?

**Comment by ambihelical** [comment]
When I tried gnome-specimen on my selected font (Ubuntu Mono Bold), it renders the same way as i3.  However, some fonts render correctly, some don't.  I suppose a workaround is a different font.

**Comment by Airblader** [comment]
This confirms that it's not an i3 issue. I think you should file this bug against pango for now and see what they say.

**Comment by ambihelical** [comment]
Update: Looking around in specimen, every bold font I've tried seems exhibit the problem.

**Comment by ambihelical** [comment]
To amend my previous comment, one bold font did seem to work, Symbola Bold. 
I won't be reporting this to pango as I've already spent too much time on it, but for anyone seeing this behavior, I found if I use a span with `font='Symbola Bold 12'` for just the special character, this works well enough, and renders much cleaner.  My suspicion is that pango or something is substituting a bitmap font without a space character for these high-code characters, and for some reason this is happening for the next character as well.
